### ISSUE TITLE
AdminClient is not getting all the config properties.

### ISSUE THREAD

**Issue by hguerrero** [quarkusio/quarkus#4081]
**Describe the bug**
AdminClient is not able to connect to SSL secured Kafka cluster

**Expected behavior**
AdminClient is configured with the same properties as KafkaStreams 

**Actual behavior**
AdminClient config is only using the bootstrap server

**To Reproduce**
1. Configure SSL connection 
2. Add passthrough config
3. Try to start KafkaStreams application

**Configuration**
```properties
# Add your application.properties here, if applicable.
kafka-streams.security.protocol=SSL
kafka-streams.ssl.keystore.location=src/main/resources/keystore.jks
kafka-streams.ssl.keystore.password=password
kafka-streams.ssl.truststore.location=src/main/resources/keystore.jks
kafka-streams.ssl.truststore.password=password
```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: 
 - Output of `java -version`: 
 - GraalVM version (if different from Java): 
 - Quarkus version or git rev: 

**Additional context**
(Add any other context about the problem here.)

**Comment by gsmet** [comment]
Maybe you could provide the error you have? That will be helpful.

Thanks.

**Comment by hguerrero** [comment]
This is the error:

```
2019-09-18 18:38:17,808 ERROR [io.qua.kaf.str.run.KafkaStreamsTopologyManager] (pool-2-thread-1) Failed to get topic names from broker: java.util.concurrent.TimeoutException
        at org.apache.kafka.common.internals.KafkaFutureImpl$SingleWaiter.await(KafkaFutureImpl.java:108)
        at org.apache.kafka.common.internals.KafkaFutureImpl.get(KafkaFutureImpl.java:272)
        at io.quarkus.kafka.streams.runtime.KafkaStreamsTopologyManager.waitForTopicsToBeCreated(KafkaStreamsTopologyManager.java:151)
        at io.quarkus.kafka.streams.runtime.KafkaStreamsTopologyManager.lambda$onStart$1(KafkaStreamsTopologyManager.java:119)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)```

**Comment by geoand** [comment]
Hi,

Does this problem only occur in native mode or can it be reproduced in regular JVM mode as well?

**Comment by hguerrero** [comment]
This is JVM mode, I haven't tried native.

**Comment by nomadgis** [comment]
+1
Quarkus 0.22.0

**Comment by jaikiran** [comment]
Perhaps this https://github.com/quarkusio/quarkus/pull/4149 will help?

**Comment by geoand** [comment]
This should now be closed via #4149
